### DISCUSSION TITLE
Send data to fancybox programmatically?

### DISCUSSION THREAD

**Discussion by LewBlu** [fancyapps/ui#284] [Q&A] [answered]
Hi,

I've read through the documentation and can't seem to find anything to do with sending data via ajax (programmatically), this was possible in previous versions. Is this still a feature, if so would someone be able to provide and example (either GET or POST is fine)?

Currently have the following:
`Fancybox.show([{
    src: '/allocate',
    type: 'ajax'
}]);`

I've tried using things such as settings and data properties from previous versions but nothing seems to be working. Any help would be much appreciated.

**Comment by fancyapps** [comment, marked as answer]
Hi,

Sorry, but I could not understand your question. What do you mean by "Send data to fancybox"? Your snippet works, so, what is missing?

**Reply by LewBlu** [reply]
So for example if I wanted to send some data with the Ajax, so that way the fancybox would be rendered with information from the data or the data can be used to fetch information from the database to be displayed in the fancybox.

Another way of thinking of it is like sending a POST request so data can be sent with the "src".

**Reply by fancyapps** [reply]
So, you simply want to send additional data to the server while making Ajax request? You can do it like this:

```js
Fancybox.show([{ src: '/allocate', type: 'ajax', ajax: "foo=bar&lorem=ipsum" }]);
```

then your $_POST data will look like this:

```
array(2) {
  ["foo"]=>
  string(3) "bar"
  ["lorem"]=>
  string(5) "ipsum"
}
```

(just make sure you are using the latest version, currently v4.0.27)
